docsite/docs/architecture/authentication.md

```diff
@@ -0,0 +1,27 @@
+# Whitefox authentication
+
+By default, whitefox runs as a insecure server.
+
+## Token authentication
+
+We provide a simple way to secure your application using a token provided as an environment variable.
+
+Two properties from application.properties are used to control the authentication methods of your instance:
```

Comment:
i would create a table with the property names and expected values and point to quakus doc about property resolution :)